src/main/java/hudson/plugins/accurev/delegates/AbstractModeDelegate.java

```diff
@@ -267,22 +299,39 @@ protected String getStreamColorStream() {
 
     private void setStreamColor() throws IOException {
         if (isSteamColorEnabled()) {
-            //For AccuRev 6.0.x versions
-            SetProperty.setproperty(scm, accurevWorkingSpace, listener, launcher, accurevEnv, server, getStreamColorStream(), getStreamColor(), "style");
-
-            //For AccuRev 6.1.x onwards
-            SetProperty.setproperty(scm, accurevWorkingSpace, listener, launcher, accurevEnv, server, getStreamColorStream(), getStreamColor(), "streamStyle");
+            SetProperty.setproperty(scm, accurevWorkingSpace, listener, launcher, accurevEnv, server, getStreamColorStream(),
+                    getStreamColor(), getStreamTypeParameter());
         }
     }
 
+     * populate the whole workspace if workspace delete option selected else populate latest transactions from the jenkins
+     * @param populateRequired
+     * @return
+     * @throws IOException
+     */
     protected boolean populate(boolean populateRequired) throws IOException {
         if (populateRequired) {
+            String stream = getPopulateStream();
+            int lastTransaction = NumberUtils.toInt(getLastBuildTransaction(), 0);
+            logger.info("Last transaction from jenkin " + lastTransaction);
+            String filePath = (lastTransaction == 0 || scm.isDeleteWorkspaceBeforeBuildStarts()) ? null : getFileRevisionsToBeIncluded(
```

Comment:
I believe Accurev already handles the use case if the folder is already populated so why is this a good idea to write a file for what your about to populate?